### DISCUSSION TITLE
Help with addfilter

### DISCUSSION THREAD

**Discussion by schm0** [staxrip/staxrip#1072] [Q&A] [answered]
Hi!
How to use the CLI addfilter option ?
I tried the following argument:
AddFilter:true,"TDecimate","Frame Rate","TDecimate(mode=7, rate=10)"
But everything after the last , isn't interpreted anymore by staxrip? ( , rate=10) is missing )

**Comment by Dendraspis** [comment, marked as answer]
Interesting, actually this is kind of a bug - the system removes the Quotes when StaxRip asks for them, so it handles "rate=10)" to be a new parameter.
I'll try to fix it for the next release...
